src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py

```diff
@@ -290,7 +294,19 @@ def update_labels(self, dirty=False):
 
         self.category_selector.setHidden(root and (discovered or personal_model))
         # initialize the channel page
-        self.channel_name_label.setText(self.model.channel_info['name'])
+
+        # Assemble the channels navigation breadcrumb by utilising RichText links feature
+        self.channel_name_label.setTextFormat(Qt.RichText)
+        breadcrumb_text = ""
+        for m, model in enumerate(self.channels_stack[:-1]):
+            breadcrumb_text += (
+                f'<a style="text-decoration:none;color:#A5A5A5;" href="{m}">{model.channel_info["name"]}</a>'
```

Comment:
Anchor `href` navigation works in Qt?

Reply:
It is possible to add RichText formatting to RichText QT text type objects. That includes hyperlinks. And one can catch hyperlink signals from clicking the links.